src/cluster/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use structopt::StructOpt;
+use fluvio_cluster::cli::ClusterOpt;
+use fluvio_future::task::run_block_on;
+
+fn main() -> color_eyre::Result<()> {
+    color_eyre::config::HookBuilder::blank()
+        .display_env_section(false)
+        .install()?;
+    let root: ClusterOpt = ClusterOpt::from_args();
```

Comment:
Without having suggestion trait, we need to duplicate logic for suggestion.   Should create common util to process error in the common extension

Reply:
See my comment above